packages/builder/src/components/design/settings/controls/ButtonActionEditor/actions/ShowNotification.svelte

```diff
@@ -47,6 +47,15 @@
   />
   <Label />
   <Checkbox text="Auto dismiss" bind:value={parameters.autoDismiss} />
+  {#if parameters.autoDismiss}
+    <Label>Duration (ms)</Label>
+    <DrawerBindableInput
+      title="Duration"
+      {bindings}
+      value={parameters.duration}
+      on:change={e => (parameters.duration = e.detail)}
```

Comment:
I think it would be useful to add a placeholder of 3000 here to let users know what the default is, if they don't customise this.